taskt/Core/Automation/Commands/IE Browser Commands/IECloseBrowserCommand.cs

```diff
@@ -1,42 +1,48 @@
-﻿using System;
+﻿using SHDocVw;
+using System;
 using System.Collections.Generic;
 using System.Windows.Forms;
 using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Automation.Engine;
+using taskt.Core.Utilities.CommonUtilities;
 using taskt.UI.CustomControls;
 using taskt.UI.Forms;
-using taskt.Core.Utilities.CommonUtilities;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("IE Browser Commands")]
-    [Attributes.ClassAttributes.Description("This command allows you to close the associated IE web browser")]
-    [Attributes.ClassAttributes.ImplementationDescription("This command implements the 'InternetExplorer' application object from SHDocVw.dll to achieve automation.")]
+    [Group("IE Browser Commands")]
+    [Description("This command closes the associated IE Web Browser.")]
     public class IECloseBrowserCommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please Enter the instance name")]
+        [PropertyDescription("Instance Name")]
+        [InputSpecification("Specify the name or variable that represents the IE Browser Instance which needs to be closed.")]
```

Comment:
To match with excel and word, please change the Input specification and Remarks to the following:

[InputSpecification("Enter the unique instance that was specified in the ** Create Browser** command.")]

 [Remarks("Failure to enter the correct instance or failure to first call ** Create Browser** command will cause an error.")]

I put a space between ** and Create to stop it from going bold. Don't include that. Please apply all these changes in all commands except the create one.

Reply:
done